README.md

```diff
@@ -51,36 +47,61 @@ $ brew install cocoapods
 
 2. Run `pod install` in project's root directory.
 
-3. Open iina.xcworkspace in the [latest public version of Xcode](https://itunes.apple.com/us/app/xcode/id497799835). *IINA may not build if you use any other version.*
+IINA ships with pre-compiled dynamic libraries for convenience reasons. If you aren't planning on modifying these libraries, you can follow the instructions below to build IINA; otherwise, skip down to [Building mpv manually](#building-mpv-manually):
+
+### Using the pre-compiled libraries
+
+1. Open iina.xcworkspace in the [latest public version of Xcode](https://itunes.apple.com/us/app/xcode/id497799835). *IINA may not build if you use any other version.*
+
+2. Build the project.
 
 ### Building mpv manually
 
 1. Build your own copy of mpv. If you're using a package manager to manage dependencies, the steps below outline the process.
 
 #### Homebrew
 
-Use our tap, since passes the correct flags during configuration:
+Use our tap as it passes in the correct flags to mpv's configure script:
 
 ```console
 $ brew tap iina/homebrew-mpv-iina
-$ brew install mpv-iina
+$ brew install mpv-iina coreutils
```

Comment:
Ok, so multiple things. For one, if this is an issue with out tap in particular we'd very much like to fix this ourselves rather than introduce workarounds in our build process. Second, even if we cannot fix this, I'm loath to asking users to install `coreutils` for something that ostensibly only requires `readlink` at most.

Reply:
I agree that if `mpv-iina` could be fixed then there's no need to install `coreutils`. Let me take another stab at the problem...

Basically, here's how `libmpv.dylib` looks out-of-the-box with `mpv`:
```console
$ ls -la $(realpath /usr/local/opt/mpv/lib/libmpv.dylib)
-rw-r--r--  1 athairus  staff    13M May 11 12:55 /usr/local/Cellar/mpv/0.29.1_3/lib/libmpv.1.101.0.dylib
```
and here's how it looks out-of-the-box with `mpv-iina`:
```console
$ ls -la $(realpath /usr/local/opt/mpv-iina/lib/libmpv.dylib)
-r--r--r--  1 athairus  admin   3.6M May 12 22:31 /usr/local/Cellar/mpv-iina/0.29.1_3/lib/libmpv.1.101.0.dylib
```

Running `other/change_lib_dependencies.rb` gives you this error:
```console
$ other/change_lib_dependencies.rb $(brew --prefix) $(brew --prefix mpv-iina)/lib/*.dylib
rm -rf /Users/athairus/Code/3rd-party/iina/deps/lib/
mkdir /Users/athairus/Code/3rd-party/iina/deps/lib/
cp -p /usr/local/Cellar/mpv-iina/0.29.1_3/lib/libmpv.1.101.0.dylib /Users/athairus/Code/3rd-party/iina/deps/lib/libmpv.1.dylib
cp -p /usr/local/Cellar/mpv-iina/0.29.1_3/lib/libmpv.1.101.0.dylib /Users/athairus/Code/3rd-party/iina/deps/lib/libmpv.1.dylib
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `initialize': Permission denied @ rb_sysopen - /Users/athairus/Code/3rd-party/iina/deps/lib/libmpv.1.dylib (Errno::EACCES)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `open'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `block in copy_file'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `open'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `copy_file'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1361:in `copy'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:473:in `block in copy_entry'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1499:in `wrap_traverse'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:470:in `copy_entry'
	from other/change_lib_dependencies.rb:91:in `block in <main>'
	from other/change_lib_dependencies.rb:81:in `each'
	from other/change_lib_dependencies.rb:81:in `<main>'
```

The reason it fails is because of it copying basically the same file twice. The second time it copies it's trying to overwrite a file that does not have +w. I kinda just realized that a better solution to this would be to explicitly specify the library to copy so it's only copying once.

```console
$ chmod +w $(realpath /usr/local/opt/mpv-iina/lib/libmpv.dylib)
$ other/change_lib_dependencies.rb $(brew --prefix) $(brew --prefix mpv-iina)/lib/libmpv.dylib
rm -rf /Users/athairus/Code/3rd-party/iina/deps/lib/
mkdir /Users/athairus/Code/3rd-party/iina/deps/lib/
cp -p /usr/local/Cellar/mpv-iina/0.29.1_3/lib/libmpv.1.101.0.dylib /Users/athairus/Code/3rd-party/iina/deps/lib/libmpv.1.dylib
=== Fix dependencies for /Users/athairus/Code/3rd-party/iina/deps/lib/libmpv.1.dylib ===
[snip]
Total 170
```

I leave it up to you folks to decide whether you should keep the wildcard option when suggesting how to invoke the script as the script only really cares about one file when the folder has several links to it:

```console
$ ls -la $(brew --prefix mpv-iina)/lib
total 7400
drwxr-xr-x   6 athairus  admin   192B May 12 22:31 .
drwxr-xr-x  10 athairus  admin   320B May 12 22:32 ..
-r--r--r--   1 athairus  admin   3.6M May 12 22:31 libmpv.1.101.0.dylib
lrwxr-xr-x   1 athairus  admin    20B May 12 22:31 libmpv.1.dylib -> libmpv.1.101.0.dylib
lrwxr-xr-x   1 athairus  admin    20B May 12 22:31 libmpv.dylib -> libmpv.1.101.0.dylib
drwxr-xr-x   3 athairus  admin    96B May 12 22:31 pkgconfig
```

Maybe someday there might be more than one actual .dylib here? Anyway, I think I'll update the CR to omit `coreutils` and have the documentation not use wildcards.

Reply:
Instead of passing `$(brew --prefix mpv-iina)/lib/*.dylib` in, what I did to compile mpv is passing `$(brew --prefix mpv-iina)/lib/libmpv.dylib`, which yields no error.

Reply:
I believe there should only be one libmpv.dylib, and I always explicitly specify the libmpv.dylib so I have never encountered this issue. Anyway, I'll mark this as resolved.